config.json

```diff
@@ -1348,6 +1348,18 @@
         "math"
       ]
     },
+    {
+      "slug": "knapsack",
+      "uuid": "ac179b77-98a5-4daf-9773-3d68b6cd8548",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 8,
+      "topics": [
+        "algorithms",
+        "arrays",
+        "control_flow_loops"
```

Comment:
This topic is not mentioned in the [topics list](https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt).
Maybe there is another topic that might fit from the official list?

Reply:
Recursion and dynamic programming would be suitable topics. But none of them are present in the file shared.